CMakeLists.txt

```diff
@@ -10,6 +10,8 @@ set (PROJECT_HOMEPAGE_URL "https://github.com/mull-project/libirm")
 
 option(LIBIRM_BUILD_32_BITS "Enable 32 bits build" OFF)
 
+find_package(LLVM REQUIRED CONFIG)
```

Comment:
This would likely break macOS builds.

Reply:
Right, I was having some build problems but this line was not part of the solution in the end. Removed.